### PR TITLE
Add total site energy use outputs

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Adds `report_simulation_output.energy_use_total_m_btu` and `report_simulation_output.energy_use_net_m_btu` columns to the results_output.csv. Allows `include_timeseries_total_consumptions` as a new input in the yaml under simulation_output_report.

## Checklist

Not all may apply:

- [x] Tests (and test files) have been updated
- [x] ~Documentation has been updated~
- [ ] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected integration/regression test changes

### CODE REVIEW

File: .github/workflows/config.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
 
       - name: Run buildstockbatch
         run: |
-          pip install git+https://github.com/NREL/buildstockbatch.git@develop
+          pip install git+https://github.com/NREL/buildstockbatch.git@total_energy_use
```
Comment (shorowit) [inline .github/workflows/config.yml:83]: Temporary change for testing the appropriate bsb.

Comment (joseph-robertson) [reply .github/workflows/config.yml:83]: Yep, change this back to `develop` after https://github.com/NREL/buildstockbatch/pull/280 is merged.

File: resources/hpxml-measures/Changelog.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ## OpenStudio-HPXML v1.4.0
 __New Features__
+- **Breaking Change**: Adds "Energy Use: Total" and "Energy Use: Net" columns to the annual results output file; allows timeseries outputs.
```
Comment (joseph-robertson) [inline resources/hpxml-measures/Changelog.md:3]: This will get changed in a future PR?

File: test/test_run_analysis.rb [hunk 3]
```diff
@@ -16,6 +16,14 @@ def before_setup
     @national_baseline = File.join(workflow_dir, 'national_baseline')
     @testing_upgrades = File.join(workflow_dir, 'testing_upgrades')
     @national_upgrades = File.join(workflow_dir, 'national_upgrades')
+
+    @expected_baseline_columns = [
```
Comment (joseph-robertson) [inline test/test_run_analysis.rb:20]: 👍

**Review by joseph-robertson — APPROVED** [review]

File: resources/hpxml-measures/Changelog.md (hunk 2 above)
Comment (shorowit) [reply resources/hpxml-measures/Changelog.md:3]: Yes

**Pull request merged**